docker/Makefile

```
@@ -0,0 +1,54 @@
+
+down: ## Stop all containers
+	docker-compose -f docker-compose.lightweight.yml down 2>/dev/null || true
+	docker-compose down 2>/dev/null || true
+
+logs: ## Show logs
+	docker-compose -f docker-compose.lightweight.yml logs -f 2>/dev/null || docker-compose logs -f
+
+status: ## Show container status
+	@docker ps --filter "name=browser-operator"
+
+chrome: ## Launch Chrome Canary with custom DevTools
```

checkmake (0.2.2)

[warning] Target body for "chrome" exceeds allowed length of 5 (6).

(maxbodylength)